test/widgets/inbox_test.dart

```diff
@@ -307,6 +307,16 @@ void main() {
       });
     });
 
+    testWidgets('empty topic', (tester) async {
+      final channel = eg.stream();
+      await setupPage(tester,
+        streams: [channel],
+        subscriptions: [(eg.subscription(channel))],
+        unreadMessages: [eg.streamMessage(stream: channel, topic: '')]);
+
+      check(find.text(eg.defaultRealmEmptyTopicDisplayName)).findsOne();
```

Comment:
nit: keep checks joined into same stanza after the setup that leads to them
```suggestion
        unreadMessages: [eg.streamMessage(stream: channel, topic: '')]);
      check(find.text(eg.defaultRealmEmptyTopicDisplayName)).findsOne();
```